ClubManagement/ClubManagement/Fragments/MoneyFragment.cs

```diff
@@ -36,9 +49,34 @@ public override View OnCreateView(LayoutInflater inflater, ViewGroup container,
 
             SetupTabView();
 
+            Init();
+
             return view;
         }
 
+        private void Init()
+        {
+            Context.DoWithAdmin(() =>
+            {
+                fabAdd.Visibility = ViewStates.Visible;
+            }, () =>
+            {
+                fabAdd.Visibility = ViewStates.Gone;
+            });
+
+            AddBudgetDialogFragment.SaveClick += (s, e) =>
```

Comment:
Should move it into contructor to avoid add event duplicate for save click

Reply:
Good comment 👍 👍 👍 👍